crates/subspace-service/Cargo.toml

```diff
@@ -16,12 +16,11 @@ include = [
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-cirrus-node-primitives = { version = "0.1.0", path = "../cirrus-node-primitives" }
+cirrus-client-executor = { path = "../../cumulus/client/cirrus-executor" }
```

Comment:
```suggestion
cirrus-client-executor = { version = "0.1.0", path = "../../cumulus/client/cirrus-executor" }
```

Reply:
I think this changed in the following PR anyway